install.sh

```diff
@@ -3,8 +3,17 @@ set -e -o pipefail
 
 # Usage:
 # ./install.sh
+# set ""
 
-VERSION=$(curl -s https://api.github.com/repos/redhat-et/microshift/releases | grep tag_name | head -n 1 | cut -d '"' -f 4)
+# ENV VARS
+# MICROSHIFT_CONFIG_ENV_ONLY=true will short-circuit the installation immediately after the host is configured, and before
+# the microshift release is downloaded.  This is to allow developers and CI to configure a host envirnment for testing
+# non-release microshift runtimes.
+MICROSHIFT_CONFIG_ENV_ONLY=${MICROSHIFT_CONFIG_ENV_ONLY:=false}
```

Comment:
Do we have a scheme for how we want to call env vars for install.sh (vs. those we pass into MicroShift)?
Just want to ensure we don't accidentally collide.

Reply:
Yes - the scheme is `MICROSHIFT_*`. Will drop the prefix to keep it cleanly separated.